addons/website_sale_stock_wishlist/static/src/js/website_sale.js

```diff
@@ -0,0 +1,68 @@
+/** @odoo-module **/
+
+import WebsiteSale from 'website_sale_stock.website_sale';
+import { is_email } from 'web.utils';
+
+WebsiteSale.include({
+    events: Object.assign({}, WebsiteSale.prototype.events, {
+        'click #add_stock_email_notification_wishlist_button': '_onClickAddStockEmailNotificationWishlistMessage',
+        'click #add_stock_email_notification_wishlist_form_button': '_onSubmitAddStockEmailNotificationWishlistForm',
+    }),
+
+    _onClickAddStockEmailNotificationWishlistMessage: function (ev) {
+        //hide the message, display the input box
+        ev.currentTarget.classList.add('d-none');
+        const formEl = ev.currentTarget.parentElement.querySelector('#add_stock_email_notification_wishlist_form');
+        formEl.classList.remove('d-none');
+    },
+
+    _onSubmitAddStockEmailNotificationWishlistForm: function (ev) {
```

Comment:
This is brand new code, why is it still using long-form function attributes and callbacks-based futures?